provider/blob/src/export/blob_exporter.rs

```diff
@@ -28,12 +27,12 @@ impl<'w> BlobExporter<'w> {
     }
 }
 
-impl DataExporter<SerializeMarker> for BlobExporter<'_> {
+impl DataExporter for BlobExporter<'_> {
```

Comment:
Discussion: Combining the traits means that we can't have structs that implement one but not the other persistence model (bakeable vs serialize). I guess we're at the point where we can make this a requirement for all data structs moving forward?

Reply:
that's what I'm thinking